state_machine/plugs/flight/lib/pycubed.py

```diff
@@ -147,289 +147,467 @@ def __init__(self):
             print(f'[ERROR][IMU] {e}\n\tMaybe try address=0x68?')
 
         # Initialize Sun Sensors
+        sun_sensors = []
+
         try:
             sun_yn = adafruit_tsl2561.TSL2561(self.i2c2, address=0x29)  # -Y
+            sun_sensors.append(sun_yn)
+        except Exception as e:
+            print('[ERROR][Sun Sensor -Y]', e)
+            
+        try:
             sun_zn = adafruit_tsl2561.TSL2561(self.i2c2, address=0x39)  # -Z
+            sun_sensors.append(sun_zn)
+        except Exception as e:
+            print('[ERROR][Sun Sensor -Z]', e)
+            
+        try:
             sun_xn = adafruit_tsl2561.TSL2561(self.i2c1, address=0x49)  # -X
-
+            sun_sensors.append(sun_xn)
+        except Exception as e:
+            print('[ERROR][Sun Sensor -X]', e)
+            
+        try:
             sun_yp = adafruit_tsl2561.TSL2561(self.i2c1, address=0x29)  # +Y
+            sun_sensors.append(sun_yp)
+        except Exception as e:
+            print('[ERROR][Sun Sensor +Y]', e)
+            
+        try:
             sun_zp = adafruit_tsl2561.TSL2561(self.i2c1, address=0x39)  # +Z
+            sun_sensors.append(sun_zp)
+        except Exception as e:
+            print('[ERROR][Sun Sensor +Z]', e)
+            
```

Comment:
Generally I like this, however there is a lot of code duplication. From my understanding we have 6 code blocks like these: 
```python
try:
    sun_yn = adafruit_tsl2561.TSL2561(self.i2c2, address=0x29)  # -Y
except Exception as e:
    print('[ERROR][Sun Sensor -Y]', e)
```

each of which only varies by the address, I think this should be done with a loop

Reply:
Yashika and I discussed this. I think we keep it as it is for now and we'll modify it once we actually know how we'll want to be accessing the sun sensors, as that is going to change while we develop the attitude determination code.

Reply:
In reality, we should have each of these initializations broken out into individual functions
- `_init_sun_sensors()`
-  `_init_coil_drivers()`
- etc.
that all get called from `__init__`.

Reply:
I like the idea of breaking down the initializations into individual functions - I can update the file to reflect this if it's a change we want now. Generally, I think it's a good idea to get all the major changes to this file done before we refactor, because pycubedmini.py is accessed in a LOT of different places.

Reply:
I think we should do this now